docs/examples/apache-zookeeper.md

```diff
@@ -36,15 +36,15 @@ zk-deploy-392770000   11s
 The statefulset defined in the `FrameworkVersion` comes up with 3 pods:
 
 ```bash
-kubectl get statefulset zk-zk
+$ kubectl get statefulset zk-zk
```

Comment:
I am always torn whether to add that to commands in documentation or not as it makes copying them more annoying. As it is a single line here, I don't believe it matters too much...

Reply:
yeah I thought about this too and there are other places in the docs where we don't use $ when there are multiple commands you can copy. It just makes it hard to distinguish command from output when both are in the same block.